tools/gromacs/editconf.xml

```diff
@@ -29,7 +32,17 @@
             <option value="gro">GRO file</option>
             <option value="pdb">PDB file</option>
         </param>
-        
+	<conditional name="renumber">
```

Comment:
Why not just the integer but making it optional?

Reply:
I had not thought about that. This seemed to work so I went with it. Let me know if you'd prefer me redoing it that way though.